packages/buefy/src/components/autocomplete/Autocomplete.vue

```diff
@@ -125,17 +128,17 @@ import CompatFallthroughMixin from '../../utils/CompatFallthroughMixin'
 import FormElementMixin from '../../utils/FormElementMixin'
 import BInput from '../input/Input.vue'
 
-type BInputComponent = InstanceType<typeof BInput>
+type BInputComponent = InstanceType<typeof BInput>;
 // `$data` interface of `Taginput` component.
 // we cannot directly import `Taginput` because it depends on `Autocomplete`.
 interface TaginputData {
-    _isTaginput: boolean
+    _isTaginput: boolean;
 }
 
 interface DataItem {
-    group?: string
+    group?: string;
```

Comment:
most of this codebase favors lines without semicolons, so I wonder why you added those, it makes it more "heterogenous". Again, it's just nitpicking here, you can ignore this if you want to